pkg/controller/seed-controller-manager/mla/datasource_grafana_controller.go

```diff
@@ -50,6 +52,9 @@ import (
 const (
 	prometheusType = "prometheus"
 	lokiType       = "loki"
+
+	// mlaAdminSettingsCRName contains a fixed name of the MLA admin settings custom resource in the cluster namespace
+	mlaAdminSettingsCRName = "mla-admin-settings"
```

Comment:
Can we move this to the resource pkg: https://github.com/kubermatic/kubermatic/blob/master/pkg/resources/resources.go#L633-L636
and call it something like `DefaultMLAAdminSettingsName`, so that we can also use it in API.